UnitTests/GitUITests/UserControls/RevisionGrid/Graph/RevisionGraphMultiThreadingTests.cs

```diff
@@ -41,7 +41,7 @@ public void Setup()
             _revisionGraph.Add(revision, RevisionNodeFlags.CheckedOut);
         }
 
-        [Test]
+        [Test, Timeout(10 /*min*/ * 60 /*s*/ * 1000 /*ms*/)]
```

Comment:
How did you come to a conclusion this test is causing hangs?
Why did you come up with 10 minutes figure?

Reply:
This test does not hang. It always takes more than 4 minutes to pass. To be on the safe side, I've set the timeout to the double time (see issue description).